.env

```diff
@@ -19,8 +19,7 @@ JWT_REFRESH_TOKEN_TTL=86400
 # IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
 #
 # DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
-# DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"
-# DATABASE_URL="postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=13&charset=utf8"
```

Comment:
just to check: why pgsql example was removed?

Reply:
Mistake of symfony recipe autoupdate, reverted!